src/main/java/com/artipie/rpm/pkg/HeaderTags.java

```diff
@@ -195,10 +195,18 @@ public String sourceRmp() {
      * Get the provide name header.
      * @return Value of header tag PROVIDENAME.
      */
-    public List<String> providers() {
+    public List<String> providerName() {
```

Comment:
@olenagerasimova Both methods have wrong names, please see above. Also, `PROVIDENAME` and `PROVIDEVERSION` constant could be changed to reflect better what we are addressing, something like `PROVIDESNAME` and `PROVIDESVERSION`.

Reply:
@paulodamaso the enum is from `org.redline_rpm` package